2021-09-04 20:21:12,894 INFO    MainThread:10004 [wandb_setup.py:_flush():69] setting env: {'entity': 'zumo09', 'project': 'flatland-rl-prova-sweep', 'sweep_id': 'y0xao9ob', 'root_dir': 'D:\\Users\\Zumo\\Documents\\UNIVERSITÀ\\AI\\DL\\Flatland_DDDQN', 'run_id': 'fxadta57', 'sweep_param_path': 'D:\\Users\\Zumo\\Documents\\UNIVERSITÀ\\AI\\DL\\Flatland_DDDQN\\wandb\\sweep-y0xao9ob\\config-fxadta57.yaml'}
2021-09-04 20:21:12,894 INFO    MainThread:10004 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '0f20b9f069a2312cc2b8e92e6f75310697d5fdfc'}
2021-09-04 20:21:12,894 INFO    MainThread:10004 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '0f20b9f069a2312cc2b8e92e6f75310697d5fdfc'}
2021-09-04 20:21:12,894 INFO    MainThread:10004 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-04 20:21:12,895 INFO    MainThread:10004 [wandb_init.py:_log_setup():342] Logging user logs to D:\Users\Zumo\Documents\UNIVERSITÀ\AI\DL\Flatland_DDDQN\wandb\run-20210904_202112-fxadta57\logs\debug.log
2021-09-04 20:21:12,895 INFO    MainThread:10004 [wandb_init.py:_log_setup():343] Logging internal logs to D:\Users\Zumo\Documents\UNIVERSITÀ\AI\DL\Flatland_DDDQN\wandb\run-20210904_202112-fxadta57\logs\debug-internal.log
2021-09-04 20:21:12,895 INFO    MainThread:10004 [wandb_init.py:init():375] calling init triggers
2021-09-04 20:21:12,895 INFO    MainThread:10004 [wandb_init.py:init():382] wandb.init called with sweep_config: {'gamma': 0.9124487551700446, 'hidden_size': 256, 'learning_rate': 0.004239067998170036, 'tau': 0.00447538688442271}
config: {'n_episodes': 50, 'env_config': 0, 'n_evaluation_episodes': 3, 'checkpoint_interval': 10, 'eps_start': 1.0, 'eps_end': 0.01, 'eps_decay': 0.99, 'buffer_size': 100000, 'buffer_min_size': 0, 'restore_replay_buffer': '', 'save_replay_buffer': False, 'batch_size': 128, 'gamma': 0.9124487551700446, 'tau': 0.00447538688442271, 'learning_rate': 0.004239067998170036, 'hidden_size': 256, 'update_every': 8, 'render': False, 'training_id': '210904202110', 'n_agents': 2, 'x_dim': 35, 'y_dim': 35, 'n_cities': 2, 'max_rails_between_cities': 2, 'max_rails_in_city': 3, 'malfunction_rate': 0.02, 'seed': 0, 'observation_tree_depth': 2, 'observation_radius': 10, 'observation_max_path_depth': 30}
2021-09-04 20:21:12,895 INFO    MainThread:10004 [wandb_init.py:init():424] starting backend
2021-09-04 20:21:12,895 INFO    MainThread:10004 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=spawn, using: spawn
2021-09-04 20:21:12,902 INFO    MainThread:10004 [backend.py:ensure_launched():135] starting backend process...
2021-09-04 20:21:12,980 INFO    MainThread:10004 [backend.py:ensure_launched():140] started backend process with pid: 11324
2021-09-04 20:21:12,981 INFO    MainThread:10004 [wandb_init.py:init():429] backend started and connected
2021-09-04 20:21:12,982 INFO    MainThread:10004 [wandb_run.py:_config_callback():843] config_cb None None {'gamma': 0.9124487551700446, 'hidden_size': 256, 'learning_rate': 0.004239067998170036, 'tau': 0.00447538688442271}
2021-09-04 20:21:12,983 INFO    MainThread:10004 [wandb_init.py:init():477] updated telemetry
2021-09-04 20:21:12,984 INFO    MainThread:10004 [wandb_init.py:init():500] communicating current version
2021-09-04 20:21:14,753 INFO    MainThread:10004 [wandb_init.py:init():505] got version response 
2021-09-04 20:21:14,753 INFO    MainThread:10004 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-09-04 20:21:15,154 INFO    MainThread:10004 [wandb_init.py:init():540] starting run threads in backend
2021-09-04 20:21:17,480 INFO    MainThread:10004 [wandb_run.py:_console_start():1601] atexit reg
2021-09-04 20:21:17,481 INFO    MainThread:10004 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-09-04 20:21:17,482 INFO    MainThread:10004 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-09-04 20:21:17,484 INFO    MainThread:10004 [wandb_run.py:_redirect():1536] Redirects installed.
2021-09-04 20:21:17,485 INFO    MainThread:10004 [wandb_init.py:init():565] run started, returning control to user process
2021-09-04 20:32:16,064 INFO    MainThread:10004 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-09-04 20:32:16,064 INFO    MainThread:10004 [wandb_run.py:_restore():1543] restore
2021-09-04 20:32:16,934 INFO    MainThread:10004 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1003
  total_bytes: 1003
}

2021-09-04 20:32:17,292 INFO    MainThread:10004 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1003
  total_bytes: 1003
}

2021-09-04 20:32:17,571 INFO    MainThread:10004 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1003
  total_bytes: 8425
}

2021-09-04 20:32:17,679 INFO    MainThread:10004 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1003
  total_bytes: 8425
}

2021-09-04 20:32:17,785 INFO    MainThread:10004 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1003
  total_bytes: 8425
}

2021-09-04 20:32:17,894 INFO    MainThread:10004 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5566
  total_bytes: 8425
}

2021-09-04 20:32:18,004 INFO    MainThread:10004 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8425
  total_bytes: 8425
}

2021-09-04 20:32:18,112 INFO    MainThread:10004 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8425
  total_bytes: 8425
}

2021-09-04 20:32:18,221 INFO    MainThread:10004 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8425
  total_bytes: 8425
}

2021-09-04 20:32:18,329 INFO    MainThread:10004 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8425
  total_bytes: 8425
}

2021-09-04 20:32:18,440 INFO    MainThread:10004 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8425
  total_bytes: 8425
}

2021-09-04 20:32:18,877 INFO    MainThread:10004 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8425
  total_bytes: 8425
}

2021-09-04 20:32:18,995 INFO    MainThread:10004 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8425
  total_bytes: 8425
}

2021-09-04 20:32:20,286 INFO    MainThread:10004 [wandb_run.py:_show_summary():1848] rendering summary
2021-09-04 20:32:20,291 INFO    MainThread:10004 [wandb_run.py:_show_history():1886] rendering history
2021-09-04 20:32:20,297 INFO    MainThread:10004 [wandb_run.py:_show_files():1915] logging synced files
